### ISSUE TITLE
Is it possible to use gevent-socketio as a socket.io client?

### ISSUE THREAD

**Issue by garyo** [abourget/gevent-socketio#254]
I need to talk with a socket.io server -- it it possible to use gevent-socketio as a client, connecting to a websocket on a server and then exchanging realtime messages?

**Comment by AusIV** [comment]
No, gevent-socketio is a socketio server, not a client. I'd recommend [socketio-client](https://pypi.org/project/socketIO-client/) for the other side of the equation.
